frontend/javascripts/dashboard/publication_card.tsx

```diff
@@ -21,26 +26,50 @@ type ExtendedDatasetDetails = APIDatasetDetails & {
 };
 const thumbnailDimension = 500;
 const miniThumbnailDimension = 75;
-const blacklistedSegmentationNames = ["2012-09-28_ex145_07x2_ROI2017"];
 
-function getDisplayName(dataset: APIDataset): string {
-  return dataset.displayName != null && dataset.displayName !== ""
-    ? dataset.displayName
-    : dataset.name;
+enum AnnotationOrDataset {
+  ANNOTATION = "ANNOTATION",
+  DATASET = "DATASET",
 }
+type AnnotationOrDatasetItem =
```

Comment:
Do we use the `-able` prefix for types somewhere already?

Reply:
> Do we use the `-able` prefix for types somewhere already?

probably not, but in the end it's an adverb which (in combination with a noun) would be perfectly acceptable to me. I don't really like the standalone version of it (`Publishable`), but wanted to avoid the `type` suffix, because we decided at some point that it's redundant in most cases. But maybe, this time it has a right to exist (especially, since you also used `-type` in your new commit).

What I don't like about the new `PublicationItemType` and `PublicationItem` is that it sounds like it could be an "item of an publication array". But `Publication != PublicationItem`. That's why I came up with `PublishableItem`. However, it's not a huge deal. Naming is hard and this discussion is probably not worth to carry out more :sweat_smile: Do what you deem is ok :)

Reply:
I think I'll keep it as is now :-)